packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -31,16 +32,17 @@ export interface PanelChromeProps {
   padding?: PanelPadding;
   title?: string;
   titleItems?: PanelChromeInfoState[];
-  menu?: React.ReactElement;
-  /** dragClass, hoverHeader, loadingState, and states not yet implemented */
+  menu?: ReactElement;
+  /** dragClass, hoverHeader not yet implemented */
   // dragClass?: string;
   hoverHeader?: boolean;
-  // loadingState?: LoadingState;
-  // states?: ReactNode[];
-  /** @deprecated in favor of prop states
+  onStreamingStop?: () => void;
+  loadingState?: LoadingState;
+  dataStateNode?: ReactNode;
```

Comment:
we don't need both of these, LoadingState has Streaming option. 

And stopping streaming action can be handled the same way as stop a normal query (via the menu)

Reply:
Sorry, think. I misunderstood dataStateNode, thought it was for the streaming state icon for some reason :)